src/providers/drpc.ts

```diff
@@ -0,0 +1,20 @@
+import { CHAIN_IDs, PUBLIC_NETWORKS } from "../constants";
+import { RPCTransport } from "./types";
+
+// Chain-specific overrides for when the Alchemy endpoint does not match the canonical chain name.
+const endpoints: { [chainId: string]: string } = {
+  [CHAIN_IDs.ARBITRUM]: "arbitrum",
+  [CHAIN_IDs.MAINNET]: "ethereum",
+  [CHAIN_IDs.SEPOLIA]: "sepolia",
```

Comment:
I think you're OK to remove the SEPOLIA definition. The endpoint names are inherited from the network definitions in https://github.com/across-protocol/constants/blob/master/src/networks.ts, so we only need to add entries here in case where the name differs from the endpoint (i.e. `Arbitrum One` vs `Arbitrum` and `Mainnet` vs `Ethereum`).

Reply:
True. https://github.com/across-protocol/sdk/pull/706/commits/5aadadf9c0a04a7cc349a53d1be42aeb40b48cd5